src/registry.ts

```diff
@@ -52,7 +51,7 @@ export class Registry {
       throw new Error('help must be string or undefined/null');
     }
 
-    if (has(this.data, `${type}.${name}`)) {
+    if (this.data[type][name]) {
```

Comment:
Reviewer note: I was wondering if `this.data[type]` could be undefined, but as there is an explicit check above (`if (['counter', 'gauge', 'histogram'].indexOf(type) === -1) {`) and the constructor initializes `this.data` with:
```
    this.data = {
      counter: {},
      gauge: {},
      histogram: {}
    };
```
it looks safe to do it like this.